### ISSUE TITLE
Error when running main.py

### ISSUE THREAD

**Issue by MayiVT** [HRNPH/AIwaifu#36]
Hello, Im trying to run these app, I've followed step by step the readme installation part, I'm using python 3.8.16 like it's said on the readme, but when I try to run the main.py, I get the following error:

```
(NeuralYui) D:\NeuralYui\AIwaifu>python main.py
Initializing... Dependencies
Loading Waifu Vocal Pipeline...
Traceback (most recent call last):
  File "main.py", line 35, in <module>
    vocal_pipeline = tts_pipeline()
  File "main.py", line 24, in __init__
    self.model = tts.OpenJtalk()
  File "D:\NeuralYui\AIwaifu\AIVoifu\tts\tts.py", line 33, in __init__
    import pyopenjtalk
  File "D:\Programs\anaconda3\envs\NeuralYui\lib\site-packages\pyopenjtalk\__init__.py", line 20, in <module>
    from .htsengine import HTSEngine
  File "pyopenjtalk/htsengine.pyx", line 1, in init pyopenjtalk.htsengine
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject

(NeuralYui) D:\NeuralYui\AIwaifu>
```

**Comment by MayiVT** [comment]
I fixed the issue installing python 3.9

**Comment by HRNPH** [comment]
seem like it's with Numpy versioning problem
